MdeModulePkg/Library/UefiBootManagerLib/BmBoot.c

```diff
@@ -2639,6 +2639,171 @@ BmEnumerateBootOptions (
   return BootOptions;
 }
 
+/* EFI System partition Scanning code Extension */
+
+EFI_BOOT_MANAGER_LOAD_OPTION * BmEspScanning (
+
+UINTN							*BootOptionCount,
+				
+OUT   EFI_BOOT_MANAGER_LOAD_OPTION			*BootOption,

```

Comment:
Please have a look at the codestyle in this file.  Your added code does not comply with EDK2 code style at all. Please reformat it @maheshtammisetti

Reply:
@miczyg1 `/home/coreboot/coreboot/DasharoModulePkg/DasharoBootPolicies/DasharoBootPolicies.inf(-1): error 000E: File/directory not found in workspace
	/home/coreboot/coreboot/IntelSiliconPkg/IntelSiliconPkg.dec`, I have updated the local repository from the main for the DasharoModulePkg, but this is a bit confusing.

Reply:
I have just built OVMF from dasharo branch using instructions from https://docs.dasharo.com/variants/qemu_q35/building-manual/

Please ensure you are following the same process

Reply:
Yes sure, I even looked at the dasharo branch and will be doing the same now